### PR TITLE
fix alignTernaryLines behavior

### PR DESCRIPTION
**Pull request by aMarCruz**
See #44

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If not an internal change) I’ve added my changes to the `CHANGELOG.unreleased.md` file following the template.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: tests/standard/jsfmt.spec.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 run_spec(__dirname, ["babel", "flow", "typescript"], {
+  arrowParens: "always",
```
Comment (brody2consult) [inline tests/standard/jsfmt.spec.js]: @aMarCruz can you explain the motivation for proposing this change?

I would probably favor making this change in a separate PR.

File: tests/standard/jsfmt.spec.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+  endOfLine: "lf",
```
Comment (brody2consult) [inline tests/standard/jsfmt.spec.js]: This change is now merged in from PR #47, should be removed from this proposal by merge or rebase.

**Review by brody2consult — COMMENTED** [review]
@aMarCruz I would like to first thank you on behalf of the entire user community for this contribution.

I will probably need a few days for a more careful review.

As I said in the line comments, it would be great if you could explain the motivation for each of the two lines you added to `tests/standard/jsfmt.spec.js`, One of the added lines is already merged in from PR #47 and should be removed from this proposal (by merge or rebase). I would favor moving the other one to its own PR as well.

File: tests/standard/jsfmt.spec.js (hunk 1 above)
Comment (aMarCruz) [reply tests/standard/jsfmt.spec.js]: ups, I thought that setting was from Standard, but it is from my own config.

**Comment by brody2consult** [comment]
Thanks @aMarCruz for the awesome work, it is now merged. I will try to release it soon.

**Comment by aMarCruz** [comment]
@brodybits , thanks!

As a side note, when using `alignTernaryLines:false`, each ternary is indented as a block using the user's prferences and avoiding the default 2-spaces indention, much like Prettier's previous behavior.
This is done even in JS fragments inside .jsx and .tsx files, as ESLint does.

This style is not so pretty but it avoids conflicts and is quite consistent.

**Comment by brody2consult** [comment]
Thanks @aMarCruz for the side note. Should we consider documenting this, or maybe just adding a comment somewhere?

**Comment by aMarCruz** [comment]
@brodybits, sure. But I don't know where and my "english" is poor.

Anyway, in the wiki of eslint-plugin-prettierx I will explain details about the behavior. I hope to have a usable version in one or two days, ...right now I'm $tarting a new project :)

**Pull request merged**